app/src/main/kotlin/org/fossify/musicplayer/playback/player/Player.kt

```diff
@@ -69,10 +70,17 @@ private fun PlaybackService.initializePlayer(
 }
 
 private fun Context.getSessionActivityIntent(): PendingIntent {
+    val intent = Intent(this, MainActivity::class.java).apply {
+
+        putExtra(EXTRA_OPEN_PLAYER, true)
+
+        flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_SINGLE_TOP
+    }
```

Comment:
Only add empty lines when they clearly improve readability. 

```suggestion
    val intent = Intent(this, MainActivity::class.java).apply {
        putExtra(EXTRA_OPEN_PLAYER, true)
        flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_SINGLE_TOP
    }
```